changelogs/api-management-service.md

```diff
@@ -1,5 +1,27 @@
 # Azure API Management service changelog
 
+# Release - API Management service: October 2023
+
+### Highlights
+
+1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
+
+### New features, fixes and improvements
+
+1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.
```

Comment:
@tomkerkhove, it looks like there are two issues here:

1) _securitywhen_ should be _security when_
2) _swagger_ should be _Swagger_

Reply:
> swagger should be Swagger

I'd argue that it should even be OpenAPI specification. Also, it's unclear from the notes how customers can use this feature flag - it would be good to explain it or point to the docs.

Reply:
@mikebudzynski, I agree with you. We should be using _OpenAPI Specification_ anywhere we use _Swagger_ going forward as we want to reference the spec, not the product implementing it. I think we have outgrown historic reasons to use _Swagger_ now.